packages/nitro-protocol/contracts/StatusManager.sol

```diff
@@ -0,0 +1,96 @@
+pragma solidity 0.7.4;
+
+import './interfaces/IStatusManager.sol';
+
+/**
+ * @dev The StatusManager is responsible for on-chain storage of the status of active channels
+ */
+contract StatusManager is IStatusManager {
+    mapping(bytes32 => bytes32) public statusOf;
```

Comment:
Should this be marked as private? Presumably, the adjudicator/asset holder will not need to interact directly with `statusOf`.

Reply:
Possibly. It is public here because it was previously public in `ForceMove`. I think another sweep will reveal a few opportunities to tighten accessors (internal to private, public to internal or private).